### DISCUSSION TITLE
Welcome to HD6303RP_EVB Discussions!

### DISCUSSION THREAD

**Discussion by TheEccentricGenius** [TheEccentricGenius/HD6303RP_EVB#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KenWillmott** [comment]
I see you put a 373 latch U7 on address lines A8-A15. Just wondering what you have in mind, because those signals are valid early in the cycle, and remain valid until the falling edge of E.
